src/scss/components/_chevron.scss

```diff
@@ -0,0 +1,7 @@
+.chevron {
```

Comment:
Again, `chevron` is a bit of a weird component name. Something like `_link-icon.scss` is a bit more transparent.